src/System.Windows.Forms/src/System/Windows/Forms/ListView.cs

```diff
@@ -5940,6 +5949,12 @@ private Font GetListHeaderFont()
         }
 
         private int GetIndexOfClickedItem()
+        {
+            var lvhi = SetupHitTestInfo();
+            return (int)User32.SendMessageW(this, (User32.WM)LVM.HITTEST, IntPtr.Zero, ref lvhi);
```

Comment:
For consistency sake, as per https://github.com/dotnet/winforms/pull/3155#discussion_r419262621
```suggestion
            return unchecked((int)(long)User32.SendMessageW(this, (User32.WM)LVM.HITTEST, IntPtr.Zero, ref lvhi));
```